chia/simulator/block_tools.py

```diff
@@ -540,6 +542,31 @@ def get_farmer_wallet_tool(self) -> WalletTool:
     def get_pool_wallet_tool(self) -> WalletTool:
         return WalletTool(self.constants, self.pool_master_sk)
 
+    # Verifies if the given plot passed any of the previous `UNIQUE_PLOTS_WINDOW` plot filters.
+    def plot_id_passed_previous_filters(
+        self, height: uint32, plot_id: bytes32, cc_sp_hash: bytes32, blocks: List[FullBlock]
```

Comment:
it looks suspicious that `height` is pass in here, rather than using the height from the `FullBlock`